### PR TITLE
upgrade to containerd 1.5.8

### PR DESCRIPTION
**Pull request by BenTheElder**
there's a stable tag with the fixes we needed now, upgrading to that.

### CODE REVIEW

**Comment by aojea** [comment]
/hold
1.22.4 ???

File: pkg/apis/config/defaults/image.go [hunk 1]
```diff
@@ -18,4 +18,4 @@ limitations under the License.
 package defaults
 
 // Image is the default for the Config.Image field, aka the default node image.
-const Image = "kindest/node:v1.22.2@sha256:9af3ab3e36fb59890b2fb3a18000930b4792d62d10d2060e0ca701e2c392d487"
+const Image = "kindest/node:v1.22.4@sha256:9871c0646a7c1d82d3a2c0d38a64ab5985b25a7716985d2ca116251efe456be8"
```
Comment (aojea) [inline pkg/apis/config/defaults/image.go:21]: 1.22.3?

Comment (aojea) [reply pkg/apis/config/defaults/image.go:21]: never mind https://github.com/kubernetes/kubernetes/releases/tag/v1.22.4

**Comment by aojea** [comment]
> /hold 1.22.4 ???

/hold cancel

https://github.com/kubernetes/kubernetes/releases/tag/v1.22.4

File: pkg/apis/config/defaults/image.go (hunk 1 above)
Comment (BenTheElder) [reply pkg/apis/config/defaults/image.go:21]: Heh, I built the packages for this yesterday to be fair 🙃

**Pull request merged**